core/container.go

```diff
@@ -446,7 +447,7 @@ func (container *Container) Build(
 	}
 
 	inputs := map[string]*pb.Definition{
-		dockerui.DefaultLocalNameContext:    contextDir.LLB,
+		dockerui.DefaultLocalNameContext:    buildctxDir.LLB,
```

Comment:
naming nit: `contextDir` should be for the `Context`, `dockerfileDir` should be the name for the `Dockerfile`.

Reply:
done. thank you for this great suggestion. 🎉